superset/assets/spec/javascripts/dashboard/reducers/dashboardState_spec.js

```diff
@@ -3,11 +3,10 @@ import { expect } from 'chai';
 
 import {
   ADD_SLICE,
-  ADD_FILTER,
+  CHANGE_FILTER,
```

Comment:
note that this was changed because `ADD_FILTER` was really used for both adding and removing filters. `REMOVE_FILTER ` was never used by anything in the dashboard app.